docs/book/v3/features/error-handling.md

```diff
@@ -195,6 +200,9 @@ You could then use a [delegator factory](container/delegator-factories.md) to
 create your logger listener and attach it to your error handler:
 
 ```php
+<?php
```

Comment:
IMO just noise, but I don't know our convention here

Reply:
It could well be and I appreciate what you're saying. However, it's a convention that I've seen in most other online documentation and examples wrt PHP for some time now.

Reply:
@settermjd 
The problem here is that we don't have a definition to mark parts of code from files.

See in the documentation of Doctrine ORM:

This example is logical:

> ```php
> <?php
> // src/Bug.php
> 
> class Bug
> {
>     public function close()
>     {
>         $this->status = "CLOSE";
>     }
> }
> ```

https://www.doctrine-project.org/projects/doctrine-orm/en/current/tutorials/getting-started.html#updating-entities

But this one is not:

> ```php
> <?php
> $product = $entityManager->getRepository('Product')
>                          ->findOneBy(array('name' => $productName));
> ```

https://www.doctrine-project.org/projects/doctrine-orm/en/current/tutorials/getting-started.html#entity-repositories

---

Therefore, any suggestion is welcome. Please add it to the [documentation repository](https://github.com/laminas/documentation/issues).
Thanks in advance! 👍🏻

Reply:
Will try and get this sorted in a day or two.

Reply:
That's where a format such as AsciiDoc would come in handy, as blocks, especially code blocks, can include a filename and path, if required. Here's a blog post that covers it nicely: https://blog.mrhaki.com/2017/01/awesome-asciidoctor-using-filename.html. 

While not native to Markdown, there is a Mkdocs extension that might be of help: [python-markdown-extensions](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title). If it is fully compatible with the Mkdocs version used to build the docs, then it might be a good solution. Thoughts, @froschdesign?